website/docs/reference/widgets/table/README.md

```diff
@@ -29,6 +29,7 @@ These properties allow you to edit the table itself. All these properties are pr
 | **Default Row Height**         | Sets the height of the row in the table - Default, short or tall.                                                                                                                                       |
 | **Enable multi-row selection** | Allows multiple rows of a table to be selected. The rows are populated in the selectedRows field.                                                                                                       |
 | **Table Record Count**         | The user has to input the total number of records returned as part of a query or API call, which will be displayed in a table. It is only visible when you choose to enable **server-side pagination**. |
+| **Allow adding a row**         | Toggles a button in the table which allows users to submit new rows of data. Only columns marked as **Editable** can accept user input. Use the **onSave** event to update the source of the table's data and reflect the user's changes. |
```

Comment:
> `the **onSave** event updates the table data source (binds to the table data property).`

I think this might be a little misleading, b/c the event doesn't actually do anything on it's own. Maybe updating the original to something like:

`Use code or a query in the **onSave** event to update the source of the table's data and reflect the user's changes.`

The biggest idea I want to make sure we communicate to user is that the "adding rows" feature with tables does not actually make any changes to the table or data. The user must actually write code/queries in the **onSave** event in order to make anything happen.

Reply:
(I made the update above, hopefully this will help make it clearer, or we can revisit later)

Reply:
@spark-c - as you pointed out (below), could you please add this information?

> The biggest idea I want to make sure we communicate to user is that the "adding rows" feature with tables does not actually make any changes to the table or data. The user must actually write code/queries in the onSave event in order to make anything happen.